tests/test_holiday_groups.py

```diff
@@ -129,3 +130,27 @@ def __init__(self, *args, **kwargs):
         test_holidays._add_visakha_bucha("Visakha Bucha")
         test_holidays._add_visakha_bucha("Visaka Bochea", KHMER_CALENDAR)
         self.assertEqual(0, len(test_holidays))
+
+
+class TestTibetanCalendarHolidays(TestCase):
+    def test_add_tibetan_calendar_holiday(self):
+        # Check for out-of-range dates.
+        class TestHolidays(HolidayBase, TibetanCalendarHolidays):
+            end_year = 2158
```

Comment:
This year is not a some special year for this calendar, so I don't see the need for this test.